### PR TITLE
Feature/api endpoints with 100% test coverage

### PR DESCRIPTION
**Pull request by stratofax**
This version of the API endpoints returns:

* A version number
* A VoteStoreID
* An empty ballot

The empty ballot is the Alameda, CA JSON file from the sample VTP election data. I copied the file to a new `data` directory, and I'm using `importlib_resources` to avoid and path issues, regardless of platform (thanks @ion-oset for that tip -- I filed it away and I knew it would come in handy!).

I've also updated the tests to cover 100% of the code. I used `pytest.fixture` to retrieve a VoteStoreID and then in turn use a valid VoteStoreID to retrieve the empty ballot.

Finally, I updated the README with instructions on how to:

- Install
- Test with uvicorn
- Test with pytest

Once this is merged onto main, @windoverwater can create a new branch to connect VTP with the web API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,79 @@
...
+
+This command will set up a virtual environment using Python 3.9. Note that the specified Python version must already be installed on your computer.
+
+This project requires Python 3.9 or later.
+
+## Run the API server
+
+1. Once the installation is complete, go to the source code directory: `cd src/vtp/web/api`
+2. Run the `uvicorn` server like this: `uvicorn main:api`
+
```
Comment (windoverwater) [inline README.md:34]: FYI - when I get to this step, it errors for me:

(vtpwebapi-py3.10) [sandy] VTP-web-api % cd src/vtp/web/api                                                                    
(vtpwebapi-py3.10) [sandy] api % uvicorn main:api                                                                              
ERROR:    Error loading ASGI app. Attribute "api" not found in module "main".

Still looking into it but wanted you to know.

Comment (stratofax) [reply README.md:34]: Oops, my bad! It's a typo on my part, the correct command is:

```bash
uvicorn main:app
```

(Not `main:api` as I typed).

I'll correct the file now. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,7 +1,85 @@
...
+## Testing the API endpoints in your browser
+
+Here are some examples of the API endpoints you can access when the uvicorn server is running. For the latest list of API endpoints, please review the code in `main.py`.
+
+### Request a Voter ID
+
+To request an empty ballot, the web client first needs to receive a VoteStoreID. This indicates that the VTP backend has created a git repository to store the voter's cast ballot. The VoteStoreID matches the voter with their vote store repository.
+
+To request a VoteStoreID, go to this endpoint: `http://127.0.0.1:8000/vote/`
+
```
Comment (windoverwater) [inline README.md:64]: FYI - entering 'http://127.0.0.1:8000/vote/' and playing around with illegal VoteStoreID's keeps crashing the chrome on my M1 mini (running the latest of everything).  Chrome version 111.0.5563.64 (Official Build) (arm64)

Comment (windoverwater) [reply README.md:64]: FWIIW it seems fine on safari

**Review by windoverwater — APPROVED** [review]
Awesome!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (stratofax) [reply README.md:64]: I've been testing on Firefox, but I'll try Chrome, too. I just updated Chrome on my Mac mini M1 as well.

[the remark this answers is not in the document]
Comment (stratofax) [reply README.md:64]: I'm not sure if I've ever had 40 tabs open, though...

**Comment by stratofax** [comment]
Squash and merge completed. I did this fast because I want @windoverwater to be able to fork a new branch off of `main` with my latest code, but @ion-oset you're still welcome and encouraged to review and comment, if you like.

**Pull request merged**